### ISSUE TITLE
[AppsFlyerTracker init] crashed on app launch. Code 0x8badf00d

### ISSUE THREAD

**Issue by recih** [AppsFlyerSDK/AppsFlyerFramework#60]
Hi, we got crash reports from Apple after app rejected, any ideas?

```
Date/Time:           2018-02-08 21:48:18.7050 +0800
Launch Time:         2018-02-08 21:47:56.5537 +0800
OS Version:          iPhone OS 11.1 (15B93)
Baseband Version:    6.17.00
Report Version:      104

Exception Type:  EXC_CRASH (SIGKILL)
Exception Codes: 0x0000000000000000, 0x0000000000000000
Exception Note:  EXC_CORPSE_NOTIFY
Termination Reason: Namespace SPRINGBOARD, Code 0x8badf00d
Termination Description: SPRINGBOARD, scene-create watchdog transgression: com.example.app exhausted real (wall clock) time allowance of 19.25 seconds |  | Elapsed total CPU time (seconds): 38.660 (user 38.660, system 0.000), 64% CPU | Elapsed application CPU time (seconds): 0.547, 1% CPU | 
Triggered by Thread:  0

Filtered syslog:
None found

Thread 0 name:  Dispatch queue: com.apple.main-thread
Thread 0 Crashed:
0   libsystem_kernel.dylib        	0x0000000181710c00 semaphore_wait_trap + 8
1   libdispatch.dylib             	0x000000018159f3d4 _dispatch_sema4_wait$VARIANT$mp + 24
2   libdispatch.dylib             	0x000000018159fd84 _dispatch_semaphore_wait_slow + 140
3   CFNetwork                     	0x000000018224fbf0 -[__NSURLBackgroundSession setupBackgroundSession] + 768
4   CFNetwork                     	0x000000018224f060 -[__NSURLBackgroundSession initWithConfiguration:delegate:delegateQueue:] + 488
5   CFNetwork                     	0x000000018235003c +[NSURLSession sessionWithConfiguration:delegate:delegateQueue:] + 608
6   app                  		0x000000010114a818 -[AppsFlyerTracker init] + 488
7   app                  		0x000000010114a128 __33+[AppsFlyerTracker sharedTracker]_block_invoke + 40
8   libdispatch.dylib             	0x000000018159d048 _dispatch_client_callout + 16
9   libdispatch.dylib             	0x00000001815a0710 dispatch_once_f$VARIANT$mp + 60
10  app                  		0x000000010114a0f8 +[AppsFlyerTracker sharedTracker] + 56
11  app                  		....
12  app                  		....
13  app                  		0x0000000100d22d88 -[AppDelegate application:didFinishLaunchingWithOptions:] (AppDelegate.mm:106)
14  UIKit                         	0x000000018b0bf9c8 -[UIApplication _handleDelegateCallbacksWithOptions:isSuspended:restoreState:] + 384
15  UIKit                         	0x000000018b2b308c -[UIApplication _callInitializationDelegatesForMainScene:transitionContext:] + 3428
16  UIKit                         	0x000000018b2b7ed8 -[UIApplication _runWithMainScene:transitionContext:completion:] + 1712
17  UIKit                         	0x000000018b54e6b4 __111-[__UICanvasLifecycleMonitor_Compatability _scheduleFirstCommitForScene:transition:firstActivation:completion:]_block_invoke + 800
18  UIKit                         	0x000000018b822c78 +[_UICanvas _enqueuePostSettingUpdateTransactionBlock:] + 160
19  UIKit                         	0x000000018b54e318 -[__UICanvasLifecycleMonitor_Compatability _scheduleFirstCommitForScene:transition:firstActivation:completion:] + 252
20  UIKit                         	0x000000018b54eb88 -[__UICanvasLifecycleMonitor_Compatability activateEventsOnly:withContext:completion:] + 748
21  UIKit                         	0x000000018bcca4b8 __82-[_UIApplicationCanvas _transitionLifecycleStateWithTransitionContext:completion:]_block_invoke + 260
22  UIKit                         	0x000000018bcca35c -[_UIApplicationCanvas _transitionLifecycleStateWithTransitionContext:completion:] + 448
23  UIKit                         	0x000000018ba3c294 __125-[_UICanvasLifecycleSettingsDiffAction performActionsForCanvas:withUpdatedScene:settingsDiff:fromSettings:transitionContext:]_block_invoke + 220
24  UIKit                         	0x000000018bbd30ac _performActionsWithDelayForTransitionContext + 112
25  UIKit                         	0x000000018ba3c144 -[_UICanvasLifecycleSettingsDiffAction performActionsForCanvas:withUpdatedScene:settingsDiff:fromSettings:transitionContext:] + 252
26  UIKit                         	0x000000018b8222d8 -[_UICanvas scene:didUpdateWithDiff:transitionContext:completion:] + 364
27  UIKit                         	0x000000018b2b661c -[UIApplication workspace:didCreateScene:withTransitionContext:completion:] + 540
28  UIKit                         	0x000000018b6c487c -[UIApplicationSceneClientAgent scene:didInitializeWithEvent:completion:] + 364
29  FrontBoardServices            	0x00000001842c5758 -[FBSSceneImpl _didCreateWithTransitionContext:completion:] + 364
30  FrontBoardServices            	0x00000001842ce060 __56-[FBSWorkspace client:handleCreateScene:withCompletion:]_block_invoke_2 + 224
31  libdispatch.dylib             	0x000000018159d048 _dispatch_client_callout + 16
32  libdispatch.dylib             	0x00000001815a46c8 _dispatch_block_invoke_direct$VARIANT$mp + 288
33  FrontBoardServices            	0x00000001842f99f4 __FBSSERIALQUEUE_IS_CALLING_OUT_TO_A_BLOCK__ + 36
34  FrontBoardServices            	0x00000001842f9698 -[FBSSerialQueue _performNext] + 404
35  FrontBoardServices            	0x00000001842f9c34 -[FBSSerialQueue _performNextFromRunLoopSource] + 56
36  CoreFoundation                	0x0000000181bc22e8 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 24
37  CoreFoundation                	0x0000000181bc2268 __CFRunLoopDoSource0 + 88
38  CoreFoundation                	0x0000000181bc1af0 __CFRunLoopDoSources0 + 204
39  CoreFoundation                	0x0000000181bbf6c8 __CFRunLoopRun + 1048
40  CoreFoundation                	0x0000000181adffb8 CFRunLoopRunSpecific + 436
41  GraphicsServices              	0x0000000183977f84 GSEventRunModal + 100
42  UIKit                         	0x000000018b0b42f4 UIApplicationMain + 208
43  seiya_iwplay                  	0x0000000100d36840 main (main.m:16)
44  libdyld.dylib                 	0x000000018160256c start + 4
```

**Comment by andr-ggn** [comment]
Hello, @recih! It's a known issue for 4.8.1 and lower versions.
It was resolved in 4.8.2.
I suggest you update to 4.8.2 or higher version
👍

**Comment by recih** [comment]
Thanks!
